src/main/java/uk/gov/register/presentation/resource/DataResource.java

```diff
@@ -50,19 +51,21 @@ public Response downloadTorrent() {
     @GET
     @Path("/feed")
     @Produces({MediaType.TEXT_HTML, MediaType.APPLICATION_JSON, ExtraMediaType.TEXT_YAML, ExtraMediaType.TEXT_CSV, ExtraMediaType.TEXT_TSV, ExtraMediaType.TEXT_TTL})
-    public EntryListView feed() {
-        return viewFactory.getEntryFeedView(
-                queryDAO.getAllEntries(ENTRY_LIMIT)
-        );
+    public EntryListView feed(@QueryParam("pageIndex") long pageIndex, @QueryParam("pageSize") long pageSize) {
```

Comment:
just that I don't immediately know how missing values will be handled.  dropwizard already ensures that `Optional` query parameters can be processed appropriately, so there won't be any extra logic.